### PR TITLE
Fixed the mk files in the 'subset_frameworks' directory

### PR DESCRIPTION
**Pull request by adil-hussain-84**
Fixed the mk files in the [subset_frameworks][2] directory by following the pattern in the [subset_frameworks/json_framework.mk][1] file. Fixes [#2165][3].

[1]: https://github.com/google/j2objc/blob/master/jre_emul/subset_frameworks/json_framework.mk
[2]: https://github.com/google/j2objc/blob/master/jre_emul/subset_frameworks
[3]: https://github.com/google/j2objc/issues/2165

### CODE REVIEW

**Comment by adil-hussain-84** [comment]
The CI Workflow Run for this pull request failed for a reason that is unrelated to the code change. See Issue #2167 for the details.

File: jre_emul/build_subset_frameworks.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # 1. cd to the j2objc/jre_emul directory
 # 2. run "./build_subset_frameworks.sh"
 #
-# To delete these frameworks, run "./build_subset_frameworks clean".
+# To delete these frameworks, run "./build_subset_frameworks.sh clean".
 
 for mk in \
   android_util_framework.mk \
```
Comment (tomball) [inline jre_emul/build_subset_frameworks.sh:23]: Please remove the android_util_framework.mk line, and delete that file. The android_util library isn't built any, and there's an error reported when this script runs this make command.

**Review by tomball — CHANGES_REQUESTED** [review]
Looks good over all, just one error to be addressed and it's ready.

File: jre_emul/build_subset_frameworks.sh (hunk 1 above)
Comment (adil-hussain-84) [reply jre_emul/build_subset_frameworks.sh]: Done.

**Review by tomball — APPROVED** [review]
LGTM

**Comment by adil-hussain-84** [comment]
Argh, this branch's CI build will fail because it doesn't contain the change which just went into master to update the CI to always build with Java 11. I'll decline this MR, rebase my changes on top of master and create a new pull request.

**Comment by tomball** [comment]
FYI, there's no need to do that in the future. Github-only files, like `.github/workflows/main.yml`, get directly merged after review, but anything that touches the project's files have to get pulled into our internal repo first and reviewed there, before getting pushed out. We have internal CIs that are more thorough than Github's (that's why we hadn't noticed that the Github CI was broken), so this wouldn't have been an issue. Regardless, thanks for your contributions!

**Comment by tomball** [comment]
Argh, you're right -- the failing CI means that this approved MR isn't being forwarded internally.

**Pull request closed without merging**